ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -165,6 +165,26 @@ export const Homepage = withUserProfile()(class extends React.Component<
     profileApi().updatePageVisits({ page: this.pageId});
   }
 
+  async syncCompliance() {
+    const complianceStatus = profileApi().syncComplianceTrainingStatus().then(result => {
+      this.setState({
+        trainingCompleted: getRegistrationTasksMap()['complianceTraining'].isComplete(result)
+      });
+    }).catch(err => {
+      this.setState({trainingCompleted: false});
+      console.error('error fetching moodle training status');
```

Comment:
Do you want to pull the error message out of `err` (here and below)?

Reply:
yes, good idea 👍